lldb/include/lldb/Target/Statistics.h

```diff
@@ -133,6 +133,8 @@ struct ConstStringStats {
 struct StatisticsOptions {
   bool summary_only = false;
   bool load_all_debug_info = false;
+  bool include_targets = false;
+  bool include_modules = false;
```

Comment:
These should default to `true` if we are going to keep the old output if we just type `statistics dump`.